src/js/backend.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+(function () {
+  var URL = 'http://localhost:8888/code-and-magick/data';
+
+  window.upload = function (data, onSuccess) {
+    var xhr = new XMLHttpRequest();
+    xhr.responseType = 'json';
+
+    xhr.addEventListener('load', function () {
+      onSuccess(xhr.response);
+    });
+
+    xhr.open('POST', URL);
+    xhr.send(data);
+  };
+
+  window.load = function (onSuccess) {
+    var xhr = new XMLHttpRequest();
+    xhr.responseType = 'json';
+
+    xhr.open('GET', URL);
+    var onError = function (message) {
+      // console.error(message);
+    };
+    xhr.addEventListener('load', function () {
+      var error;
+
+      switch (xhr.status) {
+        case 200:
+          error = '11';
```

Comment:
```suggestion
          error = '';
```
наверно ошибка здесь не нужна